pySDC/projects/parallelSDC_reloaded/allenCahn_accuracy.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec  5 11:02:39 2023
+
+Script to investigate diagonal SDC on the Allen-Cahn problem :
+
+- error VS time-step
+- error VS computation cost
+
+Note : implementation in progress ...
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pySDC.projects.parallelSDC_reloaded.utils import getParamsSDC, getParamsRK, solutionSDC, solutionExact
+
+# Problem parameters
+tEnd = 50
+pName = "ALLEN-CAHN"
+periodic = False
+pParams = {
+    "periodic": periodic,
+    "nvars": 2**11 - (not periodic),
+    "epsilon": 0.04,
+}
+
+
+def getError(uNum, uRef):
+    if uNum is None:
+        return np.inf
+    return np.linalg.norm(uRef[-1, :] - uNum[-1, :], ord=2)
+
+
+def getCost(counters):
+    nNewton, nRHS, tComp = counters
+    return 2 * nNewton + nRHS
```

Comment:
Did you check that right hand side evaluations are not counted additionally in the Newton solver?

Reply:
They are not (cf previous discussion)